src/bindings/jna/libelektra/src/main/java/org/libelektra/ReadableKey.java

```diff
@@ -394,6 +399,15 @@ public int getBaseNameSize() {
   public int getValueSize() {
     return Elektra.INSTANCE.keyGetValueSize(getPointer());
   }
+  
+  /**
+   * @return True, if the value has no key, false otherwise
+   * @throws IllegalStateException if this {@link ReadableKey} has already been released
+   */
+  public boolean isNull() {
+	// TODO #4368 how to determine no value present for key?
+	return false; // TO BE REMOVED
```

Comment:
```suggestion
	return getValueSize() == 0;
```

An empty string has size 1, because the zero-terminator is included in the size.